runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -584,10 +586,8 @@ TR::CompilationInfoPerThreadRemote::processEntry(TR_MethodToBeCompiled &entry, J
 
    
 
-      // Copy the option strings
-      size_t clientOptSize = clientOptStr.size();
-      clientOptions = new (PERSISTENT_NEW) char[clientOptSize];
-      memcpy(clientOptions, clientOptStr.data(), clientOptSize);
+      // Copy the option string
+      copyClientOptions(clientOptStr, compInfo->persistentMemory());
```

Comment:
In the future this will become `clientSession->persistentMemory()` I assume

Reply:
Yes